### DISCUSSION TITLE
About install

### DISCUSSION THREAD

**Discussion by Darox360** [deepmodeling/deepmd-kit#1426] [Q&A]
Dear DP developers,
I want to know how to solve this problem about installation.(Python version = 3.6; DeepMD-kit-version = asc-2022)
![bc92886af091e38e54dd1d7476406ff](https://user-images.githubusercontent.com/80654702/149907932-7e4f355b-4f65-45d6-8364-6bd4a91a5d3a.png)
![fe61ac5c9aeadd894b04e3ce3829386](https://user-images.githubusercontent.com/80654702/149908281-6f408d9a-9385-4a6c-9aea-022ba23ae5f5.png)

**Comment by njzjz** [comment]
The way to resolve the problem has been shown in the error message. Have you tried it?
